modules/nf-core/ctree/main.nf

```diff
@@ -0,0 +1,56 @@
+process CTREE {
+    tag "$meta.id"
+    label "process_low"
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/r-ctree_r-mobster_r-viber_r-cli_pruned:660fb0bfa4d8d4ac':
```

Comment:
Can you swap to the https link for this tool rather than oras? When you define it on Seqera Containers there is a little toggle that'll give you the https link.